utils/makeDir.js

```diff
@@ -0,0 +1,25 @@
+const fs = require('fs');
+const path = require('path');
+
+const makeDirRecursive = (pathToCreate) => {
+  const dirs = pathToCreate.split(path.sep);
```

Comment:
Do you need to validate the path at all?

Reply:
Validate how?